### PR TITLE
docs: add 'Using Shared Server For The Team'

### LINKED ISSUE #44: docs: use cases: Share data and model files for multiple users on single machine

### PR DESCRIPTION
**Pull request by efiop**
Fixes #44

Signed-off-by: Ruslan Kuprieiev <ruslan@iterative.ai>

### CODE REVIEW

File: static/docs/use-cases/using-shared-server-for-the-team.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+# Using Shared Server For The Team
```
Comment (shcheklein) [inline static/docs/use-cases/using-shared-server-for-the-team.md]: It took me a while to recall what's this about. Shared Server is not very obvious name for this scenario. Let's use a straightforward name, something like: `Multiple Data Scientists on a Single Machine`

File: static/docs/use-cases/using-shared-server-for-the-team.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+The key principle that every data science team should aim for is to make
```
Comment (shcheklein) [inline static/docs/use-cases/using-shared-server-for-the-team.md]: Let's try to start with the description of the scenario first:

It's pretty common to see that teams prefer using one single shared machine to run their experiments ...

Comment (shcheklein) [reply static/docs/use-cases/using-shared-server-for-the-team.md]: Then, we need to mention briefly, why is is happening:

* Better resource utilization - I can utilize multiple GPUs, for example
* Probably, less expensive
* Data locality
* something I am missing?

File: static/docs/use-cases/using-shared-server-for-the-team.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+everything easilly reproducibile. Git makes it easy to share the code that you
+use to process your data and share it with the others. DVC helps with building
+your data pipelines and making them easilly reproducible. One of the best
+practices for data science teams is to work off the same data. Sharing your
+data is super valuable as it enables your colleagues and you to take advantage
+of the data processing that you've already done, and understand the context
+behind results you create and surface potential caveats. In order to share
+common data, many teams setup a shared server for the whole team. With DVC,
```
Comment (shcheklein) [inline static/docs/use-cases/using-shared-server-for-the-team.md]: May be we should omit or simplify the description above? It's too long, I'm not sure it's to the point of this use case.

File: static/docs/use-cases/using-shared-server-for-the-team.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+```dvc
+    $ git pull
+    $ dvc checkout
+    $ dvc run -d clean -o processed ./process.py clean process
+    $ git add processed.dvc
+    $ git commit -m 'process clean data'
+    $ git push
+```
+
+And now you can just as easilly get his work appear in your workspace by:
```
Comment (shcheklein) [inline static/docs/use-cases/using-shared-server-for-the-team.md]: Should we mention abvout garbage collection, and that we are workin on LRU cache strategy to keep the size limited?

**Comment by shcheklein** [comment]
Good stuff. I would simplify title and intro. I'll try to come with a picture to this.

File: static/docs/use-cases/using-shared-server-for-the-team.md (hunk 4 above)
Comment (efiop) [reply static/docs/use-cases/using-shared-server-for-the-team.md]: I don't think so, it is beyond the scope for this article.

**Comment by efiop** [comment]
@shcheklein I have actually used this https://www.forbes.com/sites/quora/2017/04/04/what-are-best-practices-for-collaboration-between-data-scientists/#babcee4335ee for the introduction to this use case and its naming.

File: static/docs/use-cases/using-shared-server-for-the-team.md (hunk 1 above)
Comment (efiop) [reply static/docs/use-cases/using-shared-server-for-the-team.md]: Ok

File: static/docs/use-cases/using-shared-server-for-the-team.md (hunk 2 above)
Comment (efiop) [reply static/docs/use-cases/using-shared-server-for-the-team.md]: Ok

File: static/docs/use-cases/using-shared-server-for-the-team.md (hunk 3 above)
Comment (efiop) [reply static/docs/use-cases/using-shared-server-for-the-team.md]: Ok

**Comment by shcheklein** [comment]
👍 let's merge, and polish online

**Pull request merged**